packages/frontend-api/build.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project name="frontend_api">
+    <property name="path.openssl.executable" value="openssl"/>
+    <property name="path.defuse-key-generate.executable" value="${path.bin}/generate-defuse-key"/>
+
+    <target name="frontend-api-generate-new-keys" description="Enable frontend API for all domains.">
+        <exec executable="${path.openssl.executable}" checkreturn="true">
+            <arg value="genrsa"/>
+            <arg value="-out"/>
+            <arg value="${path.config}/frontend-api/private.key"/>
+            <arg value="2048"/>
+        </exec>
+        <exec executable="${path.openssl.executable}" checkreturn="true">
+            <arg value="rsa"/>
+            <arg value="-in"/>
+            <arg value="${path.config}/frontend-api/private.key"/>
+            <arg value="-pubout"/>
+            <arg value="-out"/>
+            <arg value="${path.config}/frontend-api/public.key"/>
+        </exec>
+        <exec executable="${path.defuse-key-generate.executable}" checkreturn="true" outputProperty="encryption_key" error="/dev/null">
+        </exec>
+
+        <echo file="${path.config}/frontend-api/.encryption_key" append="false">${encryption_key}</echo>
+    </target>
+
+    <target name="frontend-api-enable" depends="frontend-api-generate-new-keys" description="Enable frontend API for all domains.">
+        <copy file="${path.config}/packages/frontend_api.yaml.dist" tofile="${path.config}/packages/frontend_api.yaml" overwrite="true">
```

Comment:
isn't possible to put the `frontend_api.yaml.dist` file into frontend-api package?

Reply:
28b24330b4eaeebcc2d1ffce03d4d0f5a448cbab

Reply:
thanks for the fix (actually the commit is 91448745b9cbc1c392e7f16d56ddb4549dc1ea58 ;))

```
<copy file="${path.packages}/frontend-api/src/Resources/config/frontend_api.yaml.dist" tofile="${path.config}/packages/frontend_api.yaml" overwrite="true">
```

this will work only in monorepo, `path.packages` is not available in plain project-base.
I think 
```
<copy file="${path.frontend-api}/src/Resources/config/frontend_api.yaml.dist" tofile="${path.config}/packages/frontend_api.yaml" overwrite="true">
```
could be used instead

Reply:
Sorry for bad commit hash. Fix is in this (perhaps correct) commit 2a65f09bd6b7df3913837a665f091bc3a4f32cfa